ukis_pysat/members.py

```diff
@@ -11,6 +11,6 @@ class Platform(Enum):
 
 
 class Datahub(Enum):
-    file = auto()
+    File = auto()

```

Comment:
is it worth breaking compatibility for a capital letter here? @BoehnkeC are you using this already?

Reply:
Haven't used it yet.

Reply:
I know its a bit picky, but better now than later.